src/Containers/App/App.jsx

```diff
@@ -1,15 +1,17 @@
-import React, { useState, useEffect } from 'react';
-import CssBaseline from '@material-ui/core/CssBaseline';
+import React from 'react'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import Auth from '../../pages/Auth/Auth'
```

Comment:
remove from this component your component

Reply:
Done